lib/tasks/worker.rake

```diff
@@ -1,13 +1,104 @@
 # frozen_string_literal: true
 
+require("concurrent")
+
+MAX_DELAY = 5.minutes
```

Comment:
I believe max delay would be 300 seconds in GCP?

Reply:
According to this: https://cloud.google.com/pubsub/docs/reference/rest/v1/projects.subscriptions/modifyAckDeadline, the maximum deadline we can specify is 600 seconds. I initially just lessen the MAX_DELAY so I didn't have to wait that long when checking if the right delay is being set.

Reply:
👍